Backpack/src/androidTest/java/net/skyscanner/backpack/text/BpkTextTest.kt

```diff
@@ -150,20 +150,25 @@ class BpkTextTest {
       arrayOf(
         "bpkTextXxlEmphasized",
         "sans-serif-medium",
-        30,
+        32,
         null,
         fontsAccessor(BpkText.XXL, Weight.EMPHASIZED)
       ),
       arrayOf(
         "bpkTextXxxlEmphasized",
         "sans-serif-medium",
-        36,
+        40,
         null,
         fontsAccessor(BpkText.XXXL, Weight.EMPHASIZED)
       ),
       arrayOf("bpkTextXlHeavy", "sans-serif-black", 24, null, fontsAccessor(BpkText.XL, Weight.HEAVY)),
-      arrayOf("bpkTextXxlHeavy", "sans-serif-black", 30, null, fontsAccessor(BpkText.XXL, Weight.HEAVY)),
-      arrayOf("bpkTextXxxlHeavy", "sans-serif-black", 36, null, fontsAccessor(BpkText.XXXL, Weight.HEAVY))
+      arrayOf("bpkTextXxlHeavy", "sans-serif-black", 32, null, fontsAccessor(BpkText.XXL, Weight.HEAVY)),
+      arrayOf("bpkTextXxxlHeavy", "sans-serif-black", 40, null, fontsAccessor(BpkText.XXXL, Weight.HEAVY)),
+      arrayOf("bpkTextHero1", "sans-serif-medium", 120, null, fontsAccessor(BpkText.HERO1, Weight.EMPHASIZED)),
```

Comment:
I feel like at some point we need to remove thouse tests and use screenshot tests + generator tool tests instead

Reply:
Yeah we already have screenshot tests for all of these - I think I'll just remove these as they'll be a pain to maintain for all the other typography changes going forward